ROMFS/px4fmu_common/init.d/airframes/12002_steadidrone_mavrik

```diff
@@ -16,6 +16,8 @@
 #
 # @maintainer Simon Wilks <simon@uaventure.com>
 #
+# @board px4_fmu-v2_fixedwing exclude
```

Comment:
I don't think the label (fixedwing in this case) is passed into airframes generation.

Reply:
Yep, obviously it isnt